### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by davetcoleman**
@brawner

### CODE REVIEW

**Comment by mlautman** [comment]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -1,21 +1,53 @@
...
 
+In order to reproduce our setup follow the instructions on the ros2 wiki for [installing ROS2 from source on Ubuntu 18.04](https://index.ros.org/doc/ros2/Linux-Development-Setup/).
+
+TODO: After Crystal release, support building RQT2 on top of debian ROS2 packages
+
+## Build From Source
+
+### Install Dependencies
+
+    sudo apt-get install python3-vcstool python-rosdep2
```
Comment (brawner) [inline README.md]:
```suggestion
    sudo apt-get install python3-vcstool python3-rosdep2
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,53 @@
-# Install RQT2
+# RQT2
 
-## Download RQT2 repositories and dependencies
+## Install ROS2
 
+In order to reproduce our setup follow the instructions on the ros2 wiki for [installing ROS2 from source on Ubuntu 18.04](https://index.ros.org/doc/ros2/Linux-Development-Setup/).
```
Comment (brawner) [inline README.md:5]: I would add a line about where to stop, or what parts of the source install to do. Namely, we're doing the system setup part of the source install. The rest of this how-to below subsumes parts of the source install wiki.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,53 @@
...
+
+TODO: After Crystal release, support building RQT2 on top of debian ROS2 packages
+
+## Build From Source
+
+### Install Dependencies
+
+    sudo apt-get install python3-vcstool python3-rosdep2
```
Comment (brawner) [inline README.md:15]: This is already in the source install wiki. So it should either be removed or come before it

File: README.md [hunk 4]
```diff
@@ -1,21 +1,53 @@
...
+    mkdir -p ~/ros2_ws/src
     cd ~/ros2_ws
     wget https://raw.githubusercontent.com/PickNikRobotics/rqt2_setup/master/rqt2.repos
     vcs import src < rqt2.repos
     rosdep install --from-paths src --ignore-src --rosdistro bouncy -y --skip-keys "console_bridge fastcdr fastrtps libopensplice67 rti-connext-dds-5.3.1 urdfdom_headers rqt_console rqt_gui_cpp"
 
-## Build the RQT2 repositories
+### Build the RQT2 repositories
 
     colcon build
```
Comment (brawner) [inline README.md:32]: Maybe add a comment about this taking a while, and the build time can be reduced dramatically by only building up to the rqt plugins we want to use

```colcon build --packages-up-to rqt_shell rqt_py_console```

**Review by brawner — APPROVED** [review]
Approved after review changes

File: README.md (hunk 2 above)
Comment (davetcoleman) [reply README.md:5]: this is copied from Mike's email, please further improve the README in a separate PR

File: README.md (hunk 3 above)
Comment (davetcoleman) [reply README.md:15]: added note

File: README.md (hunk 4 above)
Comment (davetcoleman) [reply README.md:32]: --packages-up-to is documented below

**Pull request merged**